airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DbConverter.java

```diff
@@ -15,35 +15,19 @@
 
 import io.airbyte.commons.enums.Enums;
 import io.airbyte.commons.json.Jsons;
-import io.airbyte.config.ActorCatalog;
-import io.airbyte.config.ActorCatalogFetchEvent;
-import io.airbyte.config.ActorDefinitionResourceRequirements;
-import io.airbyte.config.DestinationConnection;
-import io.airbyte.config.DestinationOAuthParameter;
-import io.airbyte.config.FieldSelectionData;
-import io.airbyte.config.Geography;
+import io.airbyte.config.*;
```

Comment:
same comment about wildcards import

Reply:
Thanks for noticing this. Changed the work laptop and did not change these settings